schemas/theme/section.json

```diff
@@ -0,0 +1,278 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Shopify Liquid Theme Section Schema",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "$schema": { "type": "string" },
+    "name": {
+      "type": "string",
+      "description": "The section title shown in the theme editor."
+    },
+    "tag": {
+      "type": "string",
+      "description": "The HTML element to use for the section.",
+      "enum": ["article", "aside", "div", "footer", "header", "section"]
+    },
+    "class": {
+      "type": "string",
+      "description": "Additional CSS class for the section."
+    },
+    "limit": {
+      "type": "integer",
+      "description": "The number of times a section can be added to a template or section group.",
+      "minimum": 1,
+      "maximum": 2
+    },
+    "settings": {
+      "description": "Section specific settings.",
+      "$ref": "./input_settings.json"
```

Comment:
relative `$ref`! 

Easy to write, and works on _any branch_. We don't need to hardcode main :D 

![image](https://github.com/Shopify/theme-liquid-docs/assets/4990691/f3c6d44b-ae42-40d1-ad37-fe2396efc214)